src/program.ts

```diff
@@ -1,6 +1,7 @@
 import * as path from "path";
 import ts from "typescript";
 
+export const DTS_EXTENSIONS = /\.d\.(c|m)?tsx?$/;
 export const dts = ".d.ts";
```

Comment:
is this still in use now?

Reply:
Yes, it's used during generation of `.d.ts` file from TS → https://github.com/Comandeer/rollup-plugin-dts/blob/203d22334022eac109a63bbe9c00386d7e8532df/src/index.ts#L143